Tasks/AzureRmWebAppDeployment/XMLSubstituter.ps1

```diff
@@ -0,0 +1,74 @@
+﻿Param(
+  [Parameter(mandatory=$true)][string]$xmlFilePath,
+  [Parameter(mandatory=$true)][string]$tag,
+  [string]$conditionalAttributeKey,
```

Comment:
For consistency, can you add the Parameter and mandatory attributes (with false) here.